web/src/scenes/signup/signup.tsx

```diff
@@ -66,8 +68,27 @@ export function Signup() {
 
   const submit = useCallback(
     async (values: SignupFormValues) => {
-      console.log(values);
       const signup_token = localStorage.getItem("signup_token");
+      const ironcladId = crypto.randomUUID();
+      const ironClad = await ironCladActivityApi({ signerId: ironcladId });
+
+      try {
+        await new Promise(async (resolve, reject) => {
+          if (!ironClad || !ironClad.sendAcceptance) {
+            throw new Error("Cannot init ironclad");
+          }
+
+          await ironClad.sendAcceptance({
+            onSuccess: resolve,
+            onError: reject,
+          });
+        });
+      } catch (err) {
+        toast.error("Something went wrong");
```

Comment:
```suggestion
        toast.error("Error while accepting terms");
```

Reply:
We decided to rest "Something went wrong" variant in DM.